content/docs/catalogue/assets.md

```diff
@@ -0,0 +1,139 @@
+
+Here you can see all the assets which are available in your store.
+
+The asset list is divided into 6 columns:
+
+| Column            | Description                                                |
+| :---------------- | :--------------------------------------------------------- |
+| **Sub catalogue** | The catalogue this asset belongs to                        |
+| **Code**          | The code of the catalogue item which this asset belongs to |
+| **Type**          | The type of asset                                          |
+| **Manufacturer**  | The manufacturer of your asset                             |
+| **Model**         | The model number of the asset                              |
+| **Class**         | The class of the asset. ie: `Cold chain equipment`         |
+| **Category**      | The subcategory of the asset                               |
+
+![Assets page](/docs/catalogue/images/assets_page.png)
+
+The list can display a fixed number of assets per page. On the bottom left corner, you can see how many assets are currently displayed on your screen.
+
+![Page](/docs/distribution/images/os_list_showing.png)
+
+If you have more assets than the current limit, you can navigate to the other pages by tapping on the page number or using the right of left arrows (bottom right corner).
+
+![Page](/docs/distribution/images/os_list_pagenumbers.png)
+
+You can also select a different number of rows to show per page using the option at the bottom right of the page.
+
+![Rows per page](/docs/introduction/images/rows-per-page-select.png)
+
+#### Filter Assets
+
+To add a filter to the page, choose the required filter from the drop down. Multiple filters can be combined.
+
+<p align="center">
+    <img src="/docs/catalogue/images/assets_filter.png" width="425" height="250">
+</p>
+
+#### Delete Assets
+
+Assets can be selected and deleted using the toolbar action:
+
+<p align="center">
+    <img src="/docs/catalogue/images/assets_delete.png" width="425" height="380">
+</p>
+
+### Importing And Exporting
+
+#### Import
+
+Assets can be imported from a csv file using the `Import` button
+
+![Assets nav](/docs/catalogue/images/assets_import.png)
+
+This will open an import modal.
+
+<p align="center">
+    <img src="/docs/catalogue/images/assets_import_modal.png" width="750" height="420">
+</p>
+
+An example template comma-separated-value (csv) is available for download here:
+
+<p align="center">
+    <img src="/docs/catalogue/images/asset_import_template.png" width="750" height="420">
+</p>
+
+Data will need to be converted into the format of the csv template provided in order for Open mSupply to be able to process and upload this data.
+
+A csv file can be uploaded once it has been created in the example format.
+
+#### Export
+
+Asset can be exported as a csv using the `Export` button
+
+![Assets nav](/docs/catalogue/images/asset_export_button.png)
+
+### Managing status log reasons
+
+Status log reasons can be managed using this button.
```

Comment:
This line just confused me a little and I think it can be removed. Ideally the button would be shown directly under the line, but you into the button below too, and that's next to the image.